src/transformers/models/longformer/modeling_longformer.py

```diff
@@ -424,7 +424,6 @@ def _compute_global_attention_mask(input_ids, sep_token_id, before_sep_token=Tru
     return attention_mask
 
 
-# Copied from transformers.models.roberta.modeling_roberta.create_position_ids_from_input_ids
```

Comment:
This line can/should be removed because Roberta's create_position_ids is now dependent on the past/key value cache. Longformer cannot use this cache -> therefore we delete the `# Copied from ...` statement here.

Reply:
it's removed, it's not reflected in the diff ?